docs/templating/markdown.html

```diff
@@ -104,32 +104,63 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
-        <h2 class="heading-2"><span class="blue-text">Step 1:</span> Add KituraMarkdown to the Kitura App</h2>
+        <h2 class="heading-2"><span class="blue-text">Step 1:</span> Add KituraMarkdown to Project</h2>
         <p>To use Markdown from a server, we first need to <a target="_blank" href="https://github.com/IBM-Swift/Kitura-Markdown#add-dependencies"> add Kitura-Markdown to our dependencies</a>.</p>
         <div class="info">
             <p>If you don't have a server, follow our <a target="_blank" href="https://www.kitura.io/docs/getting-started/create-server.html"> Create a server</a> guide.</p>
         </div>
-        <p>Next, we need to import the KituraMarkdown package:</p>
-        <pre><code class="language-swift">import KituraMarkdown</code></pre>
+        <p>Next, we need to import the KituraMarkdown package.</p>
         <p>Open your <strong>Application.swift</strong> file:</p>
         <pre><code>open Sources/Application/Application.swift</code></pre>
+        <p>Then at the top of the file add the import statement for KituraMarkdown:</p>
+        <pre><code class="language-swift">import KituraMarkdown</code></pre>
+        <p>Now we can register the KituraMarkdown Template Engine to our router instance.</p>
         <p>Inside the <strong>postInit()</strong> function add: </p>
         <pre><code class="language-swift">router.add(templateEngine: KituraMarkdown())</code></pre>
-        <p>This tells the server that on this instance of the router we'd like to use the KituraMarkdown template engine.</p>
         <div class="info">
           <p>Kitura supports multiple template engines being registered to a single instance of a router. By default each templating engine will handle files in the <strong>./Views</strong> directory that match the file extension it supports.</p>
         </div>
-        <p>We've just added the KituraMarkdown template engine to our router, so now we can use it in a route.</p>
-        <p>Just below our other registered routes we can create a new route for KituraMarkdown:</p>
-        <pre><code class="language-swift">router.get("/book") { request, response, next in
-    try response.render("book.md", context: [:])
-}</code></pre>
-        <p>The <strong>response.render()</strong> method attempts to populate the provided template file with the provided data.</p>
-        <p>However, as we are rendering a Markdown file the <strong>context</strong> parameter is ignored.</p>
-        <p>So in our case KituraMarkdown will take the <strong>book.md</strong> file and render the Markdown in it to be viewed in a browser.</p>
-        <p>This obviously will not work as we haven't created our <strong>book.md</strong> file.</p>
         <div class="underline"></div>
-        <h2 class="heading-2"><span class="blue-text bold">Step 2:</span> Create the template file</h2>
+        <h2 class="heading-2"><span class="blue-text">Step 2:</span> Configure KituraMarkdown</h2>
```

Comment:
It is strange that we create the markdown options here then create the template file then come back to render the route. Could we mimic stencil again and set up our the complete route here then create the markdown file?